docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -435,6 +435,49 @@ incorporate the processes associated with crime reporting, the offense-trial
 pipeline, and an awareness of how the relationship between various social actors and
 the algorithm may impact the intended outcomes of a given model.
 
+
+The 4/5ths Rule: A Common Trap
```

Comment:
As we really want people to read this, perhaps we should come up with a slightly more click-baity title, like "Why you should (almost) never apply the 4/5ths rule". 

My personal preference would go to something even more general, e.g., something like: "How 'fair' should my model be?". This is a common question data scientists have, and with this phrasing they might be more likely to read the text even if they've never heard of the 4/5ths rule. (This would also provide a nice space to add more guidelines on how to translate harms into fairness metrics, in accordance with [learning objective 1.6](https://github.com/orgs/fairlearn/projects/3/views/1))

Additionally, instead of introducing this as a separate trap, an alternative position of this paragraph would be right after the explanation of the fairness constraints. In a way, blindly applying the 4/5ths rule could be seen as an example of the formalism trap and/or a portability trap. Instead of introducing it as a separate trap, we could refer to it as an example of (one of) those traps.

Reply:
Clickbaity would be something like "You'll never guess what 90% of fairness experts say about the 4/5ths rule!!!"

I've moved it up in the doc, but am still mulling a title.